doc/api/test.md

```diff
@@ -219,6 +219,46 @@ test('a test that creates asynchronous activity', (t) => {
 });
 ```
 
+## Running tests from the command line
+
+The Node.js test runner can be invoked from the command line by passing the
+[`--test`][] flag. By default, Node.js will recursively search the current
+directory for test files:
+
+```bash
+node --test
+```
+
+Alternatively, one or more paths can be provided as the final argument(s) to
+the Node.js command, as shown below.
+
+```bash
+node --test test1.js test2.mjs test/
```

Comment:
Is the trailing slash just here to make the example clear, or is it mandatory?

Reply:
It's just to make the example clear.